jekyll/_cci2/server/installation/phase-3-execution-environments.adoc

```diff
@@ -810,9 +867,23 @@ If you have not already created them, you will need an access key and secret for
 aws iam create-access-key --user-name circleci-vm-service
 ----
 
-. *Configure server*
+. *Configure server (2 options)*
 +
+When using access keys and secrets for configuring access for the VM service, there are two options.
+
+*Option 1 - Add the keys to values.yaml*
 Add the VM Service configuration to values.yaml. Details of the available configuration options can be found in the https://circleci.com/docs/2.0/server-3-operator-vm-service[VM Service] guide.
```

Comment:
@rosieyohannan , where is this link supposed to go?  Is there a server 4 version of that page?